packages/sdk/src/tests/indexing.test.ts

```diff
@@ -0,0 +1,33 @@
+import { Indexing } from '../modules/indexing/indexing';
+import { getConfig, getIndexerUUID } from './helpers/helper';
+
+describe('Indexing tests', () => {
+  let indexing: Indexing = undefined;
+
+  beforeAll(async () => {
+    indexing = new Indexing(getConfig());
+  });
+
+  test('Deploy a indexer', async () => {
+    const response = await indexing
+      .indexer(getIndexerUUID())
+      .deployIndexer('D:\\Sqd\\moonbeam-squid');
+
+    expect(response).toBeDefined();
+    expect(response.lastDeploymentId).toBeTruthy();
+    expect(response.status).toBe(5);
+    expect(response.deployment).toBeDefined();
+  });
+
+  test('Deploy a indexer with invalid path, should return error', async () => {
```

Comment:
All `ApillonModel` classes come with a built-in `get()` method, do we have that for indexers? It needs to be tested, otherwise there needs to be an override in the `Indexer` class that that method is not supported